kanban/api/views/project.py

```diff
@@ -0,0 +1,75 @@
+from django.db.models import Q
+from kanban.api.serializers import (
+    IssueSerializer,
+    ProjectListSerializer,
+    ProjectSerializer,
+    AssignIssueSerializer,
+)
+from kanban.models.issue import Issue
+from kanban.models.project import Project
+from rest_framework import viewsets
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from user_account.api.serializers import AssignUserSerializer
+from user_account.models import UserAccount
+from rest_framework import generics
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+
+
+class ProjectViewSet(viewsets.ModelViewSet):
+    queryset = Project.objects.prefetch_related("users")
+    serializer_class = ProjectListSerializer
+    detail_serializer_class = ProjectSerializer
+
+    def get_serializer_class(self):
+        if self.action == "retrieve":
```

Comment:
do you have to use magic string here? are there no consts for this?

Reply:
I can't find anything, can u use your google cheats and help me? :)

Reply:
Sounds about right